docs/pages/database-access/reference/configuration.mdx

```diff
@@ -111,6 +111,12 @@ spec:
     ...
     -----END CERTIFICATE-----
 
+  # MySQL only options.
+  mysql:
+    # The default MySQL server version reported by the Teleport Proxy Service.
+    # When this option is set, the Database Service does not try to check the MySQL server version.
```

Comment:
I think this needs a sentence about how the Database Service uses the MySQL server version and when a user would use this option. Otherwise, the negative statement, "the Database Service does not try to check..." doesn't tell us much.

Also, does this field need to be present (but an empty string) if it's unused, or can the user leave it out?

Reply:
@ptgott How about:

```yaml
# The default MySQL server version reported by the Teleport Proxy Service.
# Teleport uses that string instead of default '8.0.0-Teleport' version when reporting 
# the server version to a connecting client. When this option is not set, the Database Service will try 
# to connect to MySQL instance on startup and fetch the server version. 
# Otherwise, it will use the provided value without connecting to a database.
# In both cases MySQL server version reported to a client will be updated on the first successful 
# connection made by a user.
```

> Also, does this field need to be present (but an empty string) if it's unused, or can the user leave it out?

When unused should not be in the configuration, but if someone decide to use empty string we are defaulting to `8.0.0-Teleport` - this is the same value as we're using right now.